react/game/src/index.js

```diff
@@ -1,7 +1,52 @@
-import React from 'react';
+import React, { Component, Fragment } from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, combineReducers } from 'redux';
+import { reducer as formReducer } from 'redux-form';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
 import './scss/index.scss';
+import Login from './app/screens/Login';
 import Game from './app/screens/Game';
 
-ReactDOM.render(<Game />, document.getElementById('root'));
+const reducers = {
+  form: formReducer
+};
+
+const reducer = combineReducers(reducers);
+const store = (window.devToolsExtension ? window.devToolsExtension()(createStore) : createStore)(reducer);
+
+class App extends Component {
+  state = { loggedin: false };
+
+  setRedirect = () => {
+    this.setState({ loggedin: true });
+  };
+
+  loginProps = () => {
+    if (this.state.loggedin) return <Game />;
+    return <Login red={this.setRedirect} />;
+  };
+
+  render() {
+    return (
+      <Router basename={this.props.path}>
```

Comment:
All router behavior must be in `src/app/index.js`, then u need to import these file as `App` component and use here. In this file, u only must import `store` and connect the `App` component with these `store`